src/database.rs

```diff
@@ -513,6 +516,31 @@ impl Database {
         self.verify.as_ref().unwrap()
     }
 
+    /// Get a statement by label.
+    pub fn statement(&mut self, name: &str) -> Option<StatementRef> {
+        match self.name_result().lookup_label(name.as_bytes()) {
+            None => None,
+            Some(lookup) => Some(self.parse_result().statement(lookup.address)),
+        }
+    }
+
+    /// Export an mmp file for a given statement.
+    pub fn export(&mut self, stmt: String) {
+        time(&self.options.clone(), "export", || {
+            let parse = self.parse_result().clone();
+            let scope = self.scope_result().clone();
+            let name = self.name_result().clone();
+            let sref = self.statement(&stmt)
+                .expect(format!("Label {} did not correspond to an existing statement",
```

Comment:
Agree that using `panic!` to print operation-failed messages is less than optimal.  Will spend some time looking at cargo and other ecosystem projects for ideas; I don't think we want to be _fancy_, but there's a lot of ergonomic space to explore.